include/fmt/format.h

```diff
@@ -918,6 +918,7 @@ FMT_CONSTEXPR inline auto count_digits(uint128_t n) -> int {
 
 // It is a separate function rather than a part of count_digits to workaround
 // the lack of static constexpr in constexpr functions.
+#ifdef FMT_BUILTIN_CLZLL
```

Comment:
Please move the `#ifdef` right before the comment:

```
#ifdef FMT_BUILTIN_CLZLL
// It is a separate function rather than a part of count_digits to workaround
// the lack of static constexpr in constexpr functions.
```

Reply:
done